Makefile

```diff
@@ -41,7 +41,7 @@ endif
 # Aliases #
 ###########
 
-build: docker.build.medea
+build: build.medea docker.build.medea build.jason
```

Comment:
It's better to provide `cargo.build crate=(medea|medea-jason)` command, while `build.medea` and `build.jason` make only as aliases to `cargo.build`.

Travis CI spec should contain the full command, no aliases.